<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>608</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>294</width>
    <height>334</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>566</width>
    <height>608</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sudoku</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>566</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Αρχείο</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionClear"/>
    <addaction name="actionSave"/>
    <addaction name="actionShow_winers"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Οδηγίες</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>Νέο παιγνίδι</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Τερματισμός</string>
   </property>
   <property name="toolTip">
    <string>Τερματισμός</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Αποθήκευση</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>Σχετικά</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Καθαρισμός</string>
   </property>
  </action>
  <action name="actionShow_winers">
   <property name="text">
    <string>Προβολή νικητών</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>